packages/now-cli/test/integration.js

```diff
@@ -209,6 +209,8 @@ test.after.always(async () => {
 });
 
 test('login', async t => {
+  t.timeout(ms('10s'));
```

Comment:
Is 10 seconds a bit low for login? I'd put 30 seconds.

Reply:
Same 10 sec seems too low for email